src/RestWrite.js

```diff
@@ -1578,6 +1578,15 @@ RestWrite.prototype.runAfterSaveTrigger = function() {
       this.config,
       this.context
     )
+    .then(result => {
+      if (
```

Comment:
what do you think about the code below to keep same pattern of the other verifications?
```
if (
  result &&
  typeof result === 'object'
)
```

Reply:
Seem great, I will change that like you suggest :)